src/base/static/components/atoms/buttons.js

```diff
@@ -109,18 +108,31 @@ const Button = styled(props => {
 
     padding: props.large ? "0.5em 0.75em 0.5em 0.75em" : "0.5em",
```

Comment:
I think these units should be in terms of `rems` instead of `ems`. I think it will simplify our design language if we think of everything as a multiplier of some root font size.